services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -429,26 +616,78 @@ public static RouteOptions fromJson(String json) {
     public abstract Builder language(String language);
 
     /**
-     * The radiuses in string format that were used during the original request.
+     * The maximum distance a coordinate can be moved to snap to the road network in meters. There
+     * must be as many radiuses as there are coordinates in the request, each separated by ;.
+     * Values can be any number greater than 0 or the string unlimited.
+     * A NoSegment error is returned if no routable road is found within the radius.
      *
-     * @param radiuses radiuses values separated by comma
+     * @param radiuses a String of radius values, each separated by ;.
      * @return this builder for chaining options together
      * @since 3.0.0
+     * @deprecated use {@link #radiusesList(List)}
+     */
+    @Deprecated
+    public Builder radiuses(String radiuses) {
+      return radiusesList(ParseUtils.parseToDoubles(radiuses));
+    }
+
+    /**
+     * The maximum distance a coordinate can be moved to snap to the road network in meters. There
+     * must be as many radiuses as there are coordinates in the request.
+     * Values can be any number greater than 0 or the string unlimited.
```

Comment:
This does not accept the `unlimited`, but I think it does accept `null`? How can we phrase this better?